scipy/signal/windows/windows.py

```diff
@@ -2017,6 +2017,7 @@ def _fftautocorr(x):
         'rect', 'rectangular'): (boxcar, False),
     ('chebwin', 'cheb'): (chebwin, True),
     ('cosine', 'halfcosine'): (cosine, False),
+    ('dpss', 'slepian'): (dpss, True),
```

Comment:
I would not add `slepian` as historically there was a different function for it

https://docs.scipy.org/doc/scipy-1.2.1/reference/generated/scipy.signal.windows.slepian.html